packages/@aws-cdk/aws-kinesisfirehose-destinations/test/s3-bucket.lit.integ.snapshot/aws-cdk-firehose-delivery-stream-s3-all-properties.assets.json

```diff
@@ -14,28 +14,28 @@
         }
       }
     },
-    "3a0df356c74318460e8172175150c9d21465d77dcfdda6f7686b4b10af25db19": {
+    "45851fd5f84b5067f2232644bbf86f53d313030a3fca7b2e0282f2017629dfc1": {
       "source": {
-        "path": "/home/hallcor/work/aws-cdk-fork",
+        "path": "/Users/rdjurasaj/code/robertd/aws-cdk",
```

Comment:
dang, what is going on here! might not be anything you can change but seems weird.

Reply:
I think this is because asset staging is disabled when running integ tests and this particular tests uses a `NodejsFunction`. 

With bundling when asset staging is disabled the stage path is the source path and for a `NodejsFunction` the source path is the path where the lock file resides (so docker can access all node modules).

Reply:
https://github.com/aws/aws-cdk/blob/49b397c343b1a2581b2f4d2c098729b21b4b0290/packages/%40aws-cdk/core/lib/asset-staging.ts#L97

Reply:
perfect, thanks